packages/firebase_analytics/test/firebase_analytics_test.dart

```diff
@@ -117,36 +108,27 @@ void main() {
   });
 
   group('$FirebaseAnalytics analytics events', () {
-    FirebaseAnalytics analytics;
-
     String name;
-    Map<String, dynamic> parameters;
+    dynamic parameters;
```

Comment:
Flutter's style is to avoid dynamic variables when possible, see: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var

(same for args below)

Reply:
I can make the change, but just asking. Should this "avoiding dynamic" be applicable to test files? I'm totally for strong-typing and safe-typing, but in test files, especially this one, (ab)using `dynamic` makes test code more verbose, and sometimes even error-prone (I forgot to reset null https://github.com/flutter/plugins/pull/1075#discussion_r248042961), while this var is always asserted/expected https://github.com/flutter/plugins/blob/521a26ea4317e4ca64334f0acf0c9366ab4f0e15/packages/firebase_analytics/test/firebase_analytics_test.dart#L138

Reply:
I think we should follow the pattern of other tests and use the `isMethodCall` matcher in the test itself, rather than asserting in the call to `setMockMethodCallHandler`. See [cloud_functions unit tests](https://github.com/flutter/plugins/blob/master/packages/cloud_functions/test/cloud_functions_test.dart) for an example.

As a bonus this will eliminate the need for the `dynamic` variable here (I think).

If you update this PR could you please make sure to update pubspec.yaml/CHANGELOG and also convert over observer_test.dart.

Thanks!

Reply:
I made the change to use `isMethodCall`, but i'm sorry i not sure what to update pubspec.yaml/CHANGELOG.

Reply:
For `observer_test.dart`, I think it's good as is.